src/nylas-connection.coffee

```diff
@@ -22,7 +22,7 @@ Folder = require('./models/folder').Folder
 
 Attributes = require './models/attributes'
 
-PACKAGE_JSON = require './package.json'
+PACKAGE_JSON = require '../package.json'
```

Comment:
Note this change, which fixes the underlying issue when relatively importing package.json